.gitignore

```diff
@@ -26,3 +26,5 @@ whitelist.json
 *.ipr
 *.iws
 src/main/resources/mixins.*.json
+*.bat
```

Comment:
We have a `gradlew.bat` that we do want to keep in the repositories. You can explicitly allow it with `!gradlew.bat`. Please verify it works as expected.

Reply:
It does though gitignore doesn't cause it to "lose" already committed files.